### DISCUSSION TITLE
add  nz-popconfirm to nz-switch component

### DISCUSSION THREAD

**Discussion by lamhungypl** [NG-ZORRO/ng-zorro-antd#6930] [Q&A]
I had  a toggle with nz-switch component, handle click toggle witch `handleChange($event)`. 

```
<nz-switch
  [(ngModel)]="user.isBlocked"
  (ngModelChange)="handleChange(user, $event)"
></nz-switch>
 ```
 
 
I wanna mix nz-switch with a pop confirrm, but can not find value of nz-switch when click `nzOnConfirm` ($event = undefined)
```
<nz-switch
    [ngModel]="user.isBlocked"
    nz-popconfirm
    nzPopconfirmTitle="Are you sure block this user?"
    (nzOnConfirm)="handleBlockChange(user, $event)"
  ></nz-switch>
  ```

**Comment by wzhudev** [comment]
Hi @lamhungypl. Could you please provide a online minimum demo?

**Reply by lamhungypl** [reply]
here you go https://stackblitz.com/edit/angular-y7ysb6

I do the trick with modalService, but it should be popconfirm, I also want loading in popconfirm OK
